src/pages/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Home from './pageComponents/Home'
+
+export default () => {
+   return(
+    // <Layout>
```

Comment:
Let's not leave in any commented out code and instead just delete them. Some of the issues with committing commented out code include:

- It's difficult for others to understand why the code was commented out in the first place, and when it would be appropriate to uncomment them again
- It adds noise to reading the file, since it's code that isn't relevant to the current functionality of the code. This will especially add up over time.
- It's easy for commented out code to become out of date, since they aren't tested or linted, and it could become so outdated that the code doesn't even work anymore